### PR TITLE
Wip remove compute

### PR DESCRIPTION
**Pull request by hhuuggoo**
@scopatz 

was I too aggressive in deleting stuff?

### CODE REVIEW

File: include/gdf/cffi/functions.h [hunk 1]
```diff
@@ -34,544 +34,8 @@ int64_t gdf_ipc_parser_get_data_offset(gdf_ipc_parser_type *handle);
...
-
-gdf_error gdf_ceil_generic(gdf_column *input, gdf_column *output);
-gdf_error gdf_ceil_f32(gdf_column *input, gdf_column *output);
-gdf_error gdf_ceil_f64(gdf_column *input, gdf_column *output);
-
-gdf_error gdf_floor_generic(gdf_column *input, gdf_column *output);
-gdf_error gdf_floor_f32(gdf_column *input, gdf_column *output);
-gdf_error gdf_floor_f64(gdf_column *input, gdf_column *output);
-
-/* casting */
```
Comment (scopatz) [inline include/gdf/cffi/functions.h:244]: I can see casting going either way. It sort of is a compute in that it operates over a column. On the other hand it does seem fairly fundemental. I'm happy with removing them for now, and adding back as needed.

File: src/tests/CMakeLists.txt [hunk 2]
```diff
@@ -38,9 +38,9 @@ enable_testing()
...
 message(STATUS "******** Configuring tests ********")
 
-add_subdirectory(foo-sample)
-add_subdirectory(datetime)
-add_subdirectory(hashing)
-add_subdirectory(sqls)
+# add_subdirectory(foo-sample)
+# add_subdirectory(datetime)
+# add_subdirectory(hashing)
+# add_subdirectory(sqls)
```
Comment (scopatz) [inline src/tests/CMakeLists.txt]: can probably just delete these lines, rather than commenting them out.

**Comment by scopatz** [comment]
Overall, this seems very reasonable.

**Comment by scopatz** [comment]
I have enabled TravisCI support, if you'd like to try removing those comment lines and pushing back up, it should trigger.

**Comment by hhuuggoo** [comment]
so the issue with casting is that arrow also has casting, and libgdf does the casting on the GPU, which means we'll have to do some work on the arrow side to sort how to dispatch to different compute kernels based on data types

I don't have a problem with that, just saying it increases the scope a bit

**Comment by scopatz** [comment]
Ahh that is fair.  Do you think this is good to go in now?

**Comment by scopatz** [comment]
@brittainhard - gpuCI is not running on this PR

**Comment by pearu** [comment]
The remaining tests (there is only one, `test_ipc`) pass ok.

**Comment by scopatz** [comment]
@pearu - ahh! that is great! It looks like this has a lot of conflicts now with master. @hhuuggoo @pearu or @andersy005 - could you resolve these please?

**Comment by pearu** [comment]
LGTM, suggest merging PR to master.

**Review by pearu — APPROVED** [review]

**Comment by scopatz** [comment]
Thanks @pearu!

**Pull request merged**